core/Execute.cpp

```diff
@@ -38,10 +39,83 @@ namespace olympia
                                                                         &exe_pipe_fact_));
             }
         }
+
+        issue_queue_topology_ = olympia::coreutils::getPipeTopology(node->getParent(), "issue_queue_topology");
+        for(size_t i = 0; i < issue_queue_topology_.size(); ++i){
+            // loop through the issue_queue_topology and create a corresponding issue_queue sparta unit
+            std::string exe_units_string = "";
+            for(auto exe_unit: issue_queue_topology_[i]){
+                exe_units_string += exe_unit + ", ";
+            }
+            const std::string issue_queue_name = "iq" + std::to_string(i);
+            // naming it for the target type it supports, i.e issue_queue_alu or issue_queue_fpu
+            const std::string tgt_name = "IssueQueue_" + issue_queue_topology_[i][0].substr(0, issue_queue_topology_[i][0].size()-1);
+            //const std::string tgt_name = issue_queue_topology_[i][0].substr(0, issue_queue_topology_[i][0].size()-1);
```

Comment:
Dead code?

Reply:
Resolved in [6b0e363](https://github.com/riscv-software-src/riscv-perf-model/pull/140/commits/6b0e3633c1b7331170afda8920f596f9fd16b233)